components/articles/blocks/LatestArticles.js

```diff
@@ -16,7 +16,7 @@ const LatestArticles = ({ block, data, blocks }) => {
   // TODO: to handle situation with multiple 'featured' blocks or 'featured' blocks
   // going *after* latestArticles.
   const { featured } = blocks;
-  let nextIndex = featured.frozen ? 0 : 1;
+  let nextIndex = featured && !featured.frozen ? 1 : 0;
```

Comment:
`featured?.frozen ? 0 : 1`

Reply:
it does not seem to be the same. in my case absense of `featured` is resolved as 0 while in your case absense of `featured` would make it 1